pkgs/tools/networking/telepresence/default.nix

```diff
@@ -8,22 +8,22 @@ let
       url = "https://github.com/datawire/sshuttle.git";
       rev = "32226ff14d98d58ccad2a699e10cdfa5d86d6269";
       sha256 = "1q20lnljndwcpgqv2qrf1k0lbvxppxf98a4g5r9zd566znhcdhx3";
-      leaveDotGit = true;
     };
 
-    buildInputs = p.buildInputs ++ [ git ];
+    nativeBuildInputs = p.nativeBuildInputs ++ [ git ];
```

Comment:
how about in that case?

```suggestion
```

Reply:
setuptools_scm als supports the environment variable [SETUPTOOLS_SCM_PRETEND_VERSION](https://github.com/pypa/setuptools_scm#environment-variables).
example: https://github.com/NixOS/nixpkgs/blob/03e90ff4328b192cd19aa5a682e3e06e2f1e08f9/pkgs/development/python-modules/python-lz4/default.nix#L31